.gitignore

```diff
@@ -1 +1,2 @@
 /node_modules/
+.idea/
```

Comment:
```suggestion
```

We don't usually include IDE directories in our `.gitignore`. It's a bit of a rabbit hole where it's not clear what IDEs should be added etc.

I suggest that you add `.idea/` to your global gitignore though - I can't think of any situation where committing it is wanted.